container/Dockerfile

```diff
@@ -1,23 +1,33 @@
 FROM fedora:30
-
 ARG APACHE_MIRROR=http://www.nic.funet.fi/pub/mirrors/apache.org
 ARG SCALA_VERSION=2.12
 ARG KAFKA_VERSION=2.4.1
-ARG CONLUENT_MAJOR_VERSION=5
-ARG CONFLUENT_MINOR_VERSION=0
-ARG CONLFUENT_PATCH_VERSION=3
+ARG CONFLUENT_MAJOR_VERSION=5
+ARG CONFLUENT_MINOR_VERSION=5
+ARG CONFLUENT_PATCH_VERSION=0
 ARG CONFLUENT_RPM_REPO=https://packages.confluent.io/rpm
 
 RUN dnf install -y java-openjdk
+RUN dnf install -y python3
+RUN dnf install -y wget
+RUN dnf install -y git
```

Comment:
It'd be better to install all these at once, as one layer

Reply:
done